docs/addons/kubernetes-nginx.md

```diff
@@ -0,0 +1,259 @@
+# Kubernetes NGINX Add-on
```

Comment:
Please add this md file entry in to `docs/addons/index.md` so it gets in to list of addons.

Reply:
Done!